docker/config/local_dev.env

```diff
@@ -7,6 +7,9 @@
 # Tecken
 # ------
 
+# Gunicorn things
+GUNICORN_TIMEOUT=60
```

Comment:
This drops the `GUNICORN_TIMEOUT` in the local dev environment to 60 seconds. It reduces the number of steps it takes to simulate the gunicorn worker timeout scenario and 60 seconds is probably a better value for a local dev environment anyhow.